_locales/it/messages.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": {
```

Comment:
According to the [project's styling rules](https://github.com/XFox111/TabsAsideExtension/blob/master/CONTRIBUTING.md#style) curly braces alway must be put on a new line. Please, fix

Reply:
Ok, I'll fix it